@uportal/dashboard-carousel/.gitignore

```diff
@@ -0,0 +1,21 @@
+.DS_Store
```

Comment:
Gitignore is not needed, these excludes are also applied by top level gitignore.

Reply:
So you want this file removed?

Reply:
Yes please :slightly_smiling_face: